docs/source/AdministratorGuide/Systems/Transformation/index.rst

```diff
@@ -627,6 +627,39 @@ Actions on transformations
 * **Complete:** The transformation can be archived by the TransformationCleaningAgent. Archived means that the data produced stay, but not the entries in the TransformationDB
 * **Clean:** The transformation is cleaned by the TransformationCleaningAgent: jobs are killed and removed from WMS. Produced and stored files are removed from the Storage Elements, when "OutputDirectories" parameter is set for the transformation.
 
+------
+HowTos
+------
+
+How to derive a production
```

Comment:
This sounds like a really cool feature. I just wonder if people would look for this if they wanted to do what is described below?

Reply:
It's a feature that has been there since long time (for several years). We used it in LHCb a few times, not without pain. I realized it was not documented so I have added here the (old) LHCb doc I found.